azure-pipelines/azure-pipelines-release.yml

```diff
@@ -14,8 +14,8 @@ pr: none
 resources:
```

Comment:
nit: Following other files in this folder, this should probably be named "release.yml" or "official-release.yml"

Reply:
Fixed!